proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -411,6 +350,7 @@ message WithdrawMsgState {
     MsgWithdrawWithinBatch msg = 6 [(gogoproto.moretags) = "yaml:\"msg\""];
 }
 
+// SwapMsgState defines the state of swap message that contains state information as it is processed in the next batch(s)
```

Comment:
```suggestion
// SwapMsgState defines the state of swap message that contains state information as it is processed in the next batch or batches
```

Reply:
(maybe for all of these message descriptions we can just use the singular "next batch"; would this be accurate?)